OSDOCS-2467: Creates a list of the SRE-supported services. (#36468)
openshift/openshift-docs: osd_policy/osd-managed-resources.adoc

[id="osd-managed-resources"]

> = {product-title} supported services

Reviewer: ```suggestion
= {product-title} managed resources
```

include::modules/attributes-openshift-dedicated.adoc[]
:context: osd-managed-resources

toc::[]

[id="osd-managed-resources-overview"]
= Overview

> The following contents file covers all resources managed by Service Reliability Engineers (SRE). Customers should not attempt to modify these resources.

Reviewer: ```suggestion
The following covers all resources managed or protected by the Service Reliability Engineering Platform (SRE-P) Team. 
```

> == Full list of managed resources

Reviewer: ```suggestion
== OSD Managed Resources
```

[%collapsible]
====
[source,yaml]
----
include::https://raw.githubusercontent.com/openshift/managed-cluster-config/master/resources/managed/all-osd-resources.yaml[]
----
====

> == List of namespaces managed by OSD addons

Reviewer: ```suggestion
== OSD Addons Namespaces
```

[%collapsible]
====
[source,yaml]
----
include::https://raw.githubusercontent.com/openshift/managed-cluster-config/master/resources/addons-namespaces/main.yaml[]
----
====

> == Objects protected by validating webhooks

Reviewer: ```suggestion
== OSD Validating Webhooks
```

[%collapsible]
====
[source,json]
----

> include::https://raw.githubusercontent.com/openshift/managed-cluster-validating-webhooks/master/docs/webhooks-short.json[]

Reviewer: Let's use the full json instead:
```suggestion
include::https://raw.githubusercontent.com/openshift/managed-cluster-validating-webhooks/master/docs/webhooks.json[]
```
